src/content/web3/index.md

```diff
@@ -1,28 +1,28 @@
 ---
 title: What is Web3 and why is it important?
-description: An introduction to Web3—the next evolution of the internet—and why it matters.
+description: An introduction to Web3—the next evolution of the World Wide Web—and why it matters.
 lang: en
 sidebar: true
 ---
 
 # Introduction to Web3 {#introduction}
 
-Centralization has helped onboard billions of people to the internet and created the stable, robust infrastructure on which it lives. At the same time, a handful of centralized entities have a stronghold on large swathes of the internet, unilaterally deciding what should and should not be allowed.
+Centralization has helped onboard billions of people to the World Wide Web and created the stable, robust infrastructure on which it lives. At the same time, a handful of centralized entities have a stronghold on large swathes of the World Wide Web, unilaterally deciding what should and should not be allowed.
```

Comment:
Again, the mixed use of the terms is confusing, especially to a novice. One of the basic rules of technical writing (I'm a technical writer by profession) is to avoid mixing terms that refer to the same thing. Pick one and stick with it. Doing otherwise creates confusion.